MeatPack tests call hidden internal symbols, causing undefined-reference errors
 in shared builds.
https://bugs.gentoo.org/978220

--- a/tests/binarize/CMakeLists.txt
+++ b/tests/binarize/CMakeLists.txt
@@ -1,4 +1,7 @@
-add_executable(binarize_tests binarize_tests.cpp)
+add_executable(binarize_tests
+    binarize_tests.cpp
+    "${PROJECT_SOURCE_DIR}/src/LibBGCode/binarize/meatpack.cpp"
+)
 
 target_link_libraries(binarize_tests ${_libname}_binarize test_common)
 
